app/frontend/stylesheets/config/_variables.scss

```diff
@@ -3,9 +3,11 @@
 // Borders & Box Shadows
 
 // Colours
+
 $blue: #7f8ea8;
 $green: #85b6aa;
 
+
 // Spacing
 $padding: 0.5rem 1rem;
 
```

Comment:
There should not be an empty line after // Colours and there should only be 1 empty line between the colours and the // Spacing.
Remove linbe 6 and 10 please